pmd-tsql/src/main/java/net/sourceforge/pmd/cpd/TSqlLanguage.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cpd;
```

Comment:
```suggestion
package net.sourceforge.pmd.lang.tsql.cpd;
```

It's probably not correct in the howto. But we are having trouble when every module defines the same package :) So, for new languages, we can fix this already.

Please move this into `net/sourceforge/pmd/lang/tsql/cpd`.